packages/workspace/src/tasks-runner/task-orchestrator.ts

```diff
@@ -291,6 +291,7 @@ export class TaskOrchestrator {
       ...envsFromFiles,
       FORCE_COLOR: forceColor,
       ...process.env,
+      NX_CACHE_KEY: task.hash,
```

Comment:
Could you rename into NX_TASK_HASH? We don't refer to it as cacheKey elsewhere, so I think it will be more consistent

Reply:
For sure!